subprojects/internal-testing/internal-testing.gradle.kts

```diff
@@ -39,6 +39,10 @@ dependencies {
     testLibraries("jmock").forEach { compile(it) }
     compile(testLibrary("spock"))
     compile(testLibrary("jsoup"))
+
+    components {
```

Comment:
Why not do this globally in DependenciesMetadataRulesPlugin? Then you only have to touch one file.

Reply:
That would indeed be easier. I'll address it in a subsequent PR so @lptr can base his work on #6324 on this branch.